docs/src/proposals/vote_subcommittee.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Vote Subcommittee
+---
+
+## Problem
+
+Each additional validator adds 1 vote per slot, increasing the
+message and compute load on the network.
+
+## Solution Overview
+
+Under the assumption that Turbine is able to guarantee that all
+honest nodes receive the block as long as at least X honest nodes
+receive the block, it is possible to allow a sampled set of validators
+to vote on that block as opposed to all the validators, and achieve
+the same liveness properties. If the network is tolerant of 1/3
+failures, then for a large enough random sample, like 200 or more,
+the probability of containing 2/3+ faulty nodes is 1:10^40. Assuming
+at least 1/3+ honest nodes are present in the subcommittee, then
+faulty forks will not get finality, and network will be able to
+continue.
+
+## Detailed Solution
+
+The following sections provide more details of the design.
+
+### Definitions
+
+* Voting Subcommittee: the set of nodes currently voting on blocks
+
+* Voting epoch: the number of slots that the voting subcommittee
+is voting for. This is separate from the leader schedule epoch.
+
+* primary subcommittee: The half of the voting subcommittee that
+is scheduled for its second epoch.
+
+* secondary subcommittee: The half of the voting subcommittee that
+is scheduled for its first epoch.
+
+* rotation seed: The seed used to generate the random sample of
+nodes. `slow_hash(penultimate snapshot hash, voting epoch start slot)`
+
+* `slow_hash`: repeated sha256 for rounds equal to 1ms on modern
+sha-ni hardware.
```

Comment:
Why the voting subcommittee for each slot of an epoch (all 432_000 slots) cannot be picked at the same time the leader schedule for that epoch is populated?

Reply:
there is more concern about grinding.  is it possible to stuff the stake weight such that attacker can get 2/3+ in any single one subcommittee?